packages/suite/src/components/suite/FormattedNftAmount.tsx

```diff
@@ -1,89 +1,127 @@
+        const tokens = transfer.multiTokenValues;
+
+        return (
+            <HiddenPlaceholder>
+                <Column alignItems={alignMultitoken}>
+                    {tokens?.map((token, index) => (
+                        <Row key={`${token.id}-${index}`} gap={4}>
+                            <Row>
+                                {signValue ? <Sign value={signValue} /> : null}
+                                {transfer.name ? (
+                                    <BlurUrls
+                                        text={translationString('TR_COLLECTION_NAME_OF_TOKEN_ID', {
+                                            tokenValue: token.value,
+                                            collectionName: transfer.name,
+                                        })}
+                                    />
+                                ) : (
+                                    <>
+                                        {token.value}x&nbsp;
+                                        <Translation id="TR_TOKEN_ID" />:
+                                    </>
+                                )}
+                            </Row>
+                            <Row>
+                                {isWithLink && network?.networkType === 'ethereum' ? (
+                                    <TrezorLink
+                                        href={`${network?.explorer.nft}/${transfer.contract}/${token.id}`}
+                                        color={theme.textSecondaryHighlight}
+                                        variant="underline"
+                                        typographyStyle="label"
+                                    >
+                                        <Id id={token.id} />
+                                    </TrezorLink>
+                                ) : (
+                                    <Id id={token.id} />
+                                )}
+                            </Row>
+                        </Row>
+                    ))}
+                </Column>
+            </HiddenPlaceholder>
+        );
+    }
 
     return (
         <HiddenPlaceholder>
-            <Container className={className}>
+            <Flex className={className}>
                 {signValue ? <Sign value={signValue} /> : null}
-                ID:&nbsp;
+                <Translation id="TR_TOKEN_ID" />
+                :&nbsp;
                 {isWithLink ? (
-                    <StyledTrezorLink href={explorerUrl}>
-                        {idComponent}
-                        {symbolComponent}
-                    </StyledTrezorLink>
+                    <TrezorLink
+                        href={explorerUrl}
+                        color={theme.textSecondaryHighlight}
+                        variant="underline"
+                        typographyStyle="label"
+                        gap="zero"
+                    >
+                        <Id id={id} />
+                        &nbsp; {symbolComponent}
```

Comment:
This seems to me weird. You set gap to zero and then you use `&nbsp;` probably for having a gap. Why?

Btw I noticed that you use &nbsp; 5x in this PR, not sure if it's not overused.

Reply:
In this case it has to be there, to preserve the underline between these components, otherwise it's being removed and there's a weird gap that is not underlined, even tho the link is the same

I didn't come up with a less hacky solution 😅

Reply:
> I realized I can do the same thing without modifying gap in the link component ![image](https://private-user-images.githubusercontent.com/101045932/391494030-8685000b-84f7-4bed-9545-307bab544c57.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMxMzQ5ODQsIm5iZiI6MTczMzEzNDY4NCwicGF0aCI6Ii8xMDEwNDU5MzIvMzkxNDk0MDMwLTg2ODUwMDBiLTg0ZjctNGJlZC05NTQ1LTMwN2JhYjU0NGM1Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMlQxMDE4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTEyZjE4ZGNiNDVjM2M4NDlkZGRiMzExMDQ5YzFjYmRjNzI0ZjFmYzYzODA5MjU4ZDQ2NDJmOGE3MmZmNzg4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.RXyrDlSh_apw2tXW4VFE5w_i0ausI7ZHUo_YIu_rTbk)

re